packages/lesswrong/server/migrations/2020-05-13-noIndexLowKarma.ts

```diff
@@ -0,0 +1,55 @@
+import { registerMigration, forEachDocumentBatchInCollection } from './migrationUtils';
+import Posts from '../../lib/collections/posts/collection';
+import moment from 'moment'
+import { forumTypeSetting } from '../../lib/instanceSettings';
+
+export const LOW_KARMA_THRESHOLD = 5
+
+const dateFormat = 'YYYY-MM-DD'
+const launchDateByForum = {
+  LessWrong: 'TODO', // lw-look-here
```

Comment:
2017-something-something

Reply:
Wow, were reasonable posts getting <5 karma on old-lw?

Reply:
No. But there are posts which were imported from other places and back-dated to when they were posted in those places, so they never appeared in the latest posts list at all.